lookup_plugins/retrieve_conjur_variable.py

```diff
@@ -14,26 +14,31 @@
 
 
 class Token:
-    def __init__(self, http_connection, id, api_key, account):
+    def __init__(self, http_connection, id, api_key, account, version):
```

Comment:
What do you think about defaulting `version` to 5?  It would mean we wouldn't make any changes moving forward, but would enable a 4.9 version